core/domain/acl_decorators.py

```diff
@@ -224,6 +351,17 @@ def can_upload_exploration(handler):
     """Decorator that checks if the current user can upload exploration."""
 
     def test_can_upload(self, **kwargs):
+        """Check of the user can upload exploration.
+
+        Returns:
+            bool. True, if teh user can upload exploration.
```

Comment:
This way it would be more clear (and consistent).
https://github.com/oppia/oppia/blob/develop/core/domain/rights_manager.py#L162

Reply:
Done :+1: